backend/molgenis-emx2-semantics/src/main/java/org/molgenis/emx2/fairdatapoint/FAIRDataPoint.java

```diff
@@ -39,15 +39,20 @@ public class FAIRDataPoint {
    * @throws Exception
    */
   public FAIRDataPoint(Request request, Schema... schemas) {
+    SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+    Date currentDateTime = new Date(System.currentTimeMillis());
```

Comment:
a classic timing stuff, in general a would suggest to use constructor injection and avoid all new .. calls instead for dedicated factory classes. But as this is done thought-out the code base using the setter  seems like a oke solution